taskfiles/boost/test_boost.cpp

```diff
@@ -0,0 +1,163 @@
+#include <chrono>
+#include <exception>
+#include <future>
+#include <iostream>
+#include <string>
+#include <string_view>
+
+#include <boost/any/bad_any_cast.hpp>
+#include <boost/asio/io_context.hpp>
+#include <boost/asio/use_future.hpp>
+#include <boost/filesystem/path.hpp>
+#include <boost/iostreams/device/back_inserter.hpp>
+#include <boost/iostreams/filtering_stream.hpp>
+#include <boost/process/v2/process.hpp>
+#include <boost/program_options/errors.hpp>
+#include <boost/program_options/options_description.hpp>
+#include <boost/program_options/parsers.hpp>
+#include <boost/program_options/value_semantic.hpp>
+#include <boost/program_options/variables_map.hpp>
+#include <boost/regex/v5/regex.hpp>  // IWYU pragma: keep
+#include <boost/regex/v5/regex_match.hpp>
+#include <boost/system/result.hpp>
+#include <boost/url/parse.hpp>
+#include <boost/url/url_view.hpp>
```

Comment:
We'll drop `misc-include-cleaner` down to a warning soon, so you can use the headers recommended by boost or just remove the pragma. Eventually, we'll likely switch to a facade wrapper for boost headers.